apps/ewallet/test/ewallet/web/orchestrator_test.exs

```diff
@@ -65,6 +67,216 @@ defmodule EWallet.Web.OrchestratorTest do
       assert error == :query_field_not_allowed
       assert params == [field_name: "status"]
     end
+
```

Comment:
Can you also test when `page` and `start_after` or `start_by` are used together and should return error? (noticed you have it in the `PaginatorTest` but since these are request attrs, might just go here as well as integration tests.